<?xml version="1.0" encoding="UTF-8"?>
<launch>

<arg name="model_path" value="/home/wms/object_segmentation/mmseg_ws/src/mmseg_deploy/checkpoints/pspnet_r50-d8_512x1024_40k_cityscapes_20200605_003338-2966598c.pt"/>
<arg name="cfg_path" value="/home/wms/object_segmentation/mmseg_ws/src/mmseg_deploy/config/cfg.yaml"/>
<arg name="verbose" value= "true"/>
<arg name="input_image" value="/rgb_cam_01/image_rect_color_sync"/>

<arg name="output_argmax" value= "/mmseg/argmax"/>
<arg name="output_color" value= "/mmseg/color_mask"/>

<!-- rosbag play -->
<include file="$(find mmseg_deploy)/launch/play_rosbag.launch" />


<!-- Launch ROS Node for net handler with all the desired parameters-->
<node pkg="mmseg_deploy" type="mmseg_node" name="mmseg_node"  output="screen">
<!-- Config -->
<param name="/model_path" value="$(arg model_path)"/>
<param name="/cfg_path" value="$(arg cfg_path)"/>
<param name="/verbose" value="$(arg verbose)"/>
<param name="/input_image" value="$(arg input_image)"/>
<param name="/output_argmax" value="$(arg output_argmax)"/>
<param name="/output_color" value="$(arg output_color)"/>

</node>

  
</launch>
